miniconda-cuda/Dockerfile

```
@@ -89,15 +89,18 @@ RUN wget --quiet ${MINICONDA_URL} -O /miniconda.sh \
     && /bin/bash /miniconda.sh -b -p /opt/conda \
     && rm -f /miniconda.sh \
     && echo "conda ${CONDA_VER}" >> /opt/conda/conda-meta/pinned \
-    && /opt/conda/bin/conda clean -tipsy \
     && ln -s /opt/conda/etc/profile.d/conda.sh /etc/profile.d/conda.sh \
     && echo ". /opt/conda/etc/profile.d/conda.sh" >> ~/.bashrc \
     && echo "conda activate base" >> ~/.bashrc \
     && echo ". /opt/conda/etc/profile.d/conda.sh" >> /etc/skel/.bashrc \
     && echo "auto_update_conda: False" >> /opt/conda/.condarc \
-    && chmod -R ugo+w /opt/conda \
     && ln -s /opt/conda /conda
 
+# Clean up conda and set permissions for all users
+RUN chmod -R ugo+w /opt/conda\
```

```suggestion
RUN chmod -R ugo+w /opt/conda \
```

not sure if this is necessary, but i think it helps readability